unified-codes/tools/scripts/dgraph/data/v2/schema.gql

```diff
@@ -0,0 +1,83 @@
+type Property {
+  type
+  value
+}
+
+type Category {
+  code
+  name
+  children
+  properties
+}
+
+type Product {
+  code
+  name
+  combines
+  children
+  properties
+}
+
+type Route {
+  code
+  name
+  children
+  properties
+}
+
+type DoseForm {
+  code
+  name
+  children
+  properties
+}
+
+type DoseQualification {
+  code
+  name
+  children
+  properties
+}
+
+type DoseStrength {
+  code
+  name
+  children
+  properties
+}
+
+type DoseUnit {
+  code
+  name
+  children
+  properties
+}
```

Comment:
```
```

export interface IUnitEntity extends IEntity {
  type: EEntityType.Unit;
  children?: IUnitEntityChildren
  properties?: IUnitEntityProperties;
}
```

Oops, this is more up-to-date than the types, will need to split `UnitEntity` into `DoseStrengthEntity` and `DoseUnitEntity`?

Reply:
Yeah I think so! I had a question for Rowena on this one actually though as it seems the current spreadsheet has some missing codes for strength and/or unit although there is text defined.

Reply:
Just a small update on this, sounds like the missing codes are just a case of the spreadsheet being still in progress (which should be ready end of this week). Still valid structure woo 🥳